static/js/components/AutoComplete.js

```diff
@@ -341,10 +350,9 @@ class AutoComplete extends Component {
     if (searchText === this.state.searchText) {
       return;
     }
-
     this.setState({
       searchText: searchText,
-      open: true,
+      open: this.menuShouldShow(searchText),
```

Comment:
We have a select field type (just fields of study right now) where we don't want to show the full menu of options if the input is blank. `menuShouldShow` is a helper function to tell us whether or not the menu should show given the searchText and component state